components/css/partials/_quote.scss

```diff
@@ -60,9 +60,9 @@ blockquote, .quote {
         padding-left: 6rem;
 
         &::before {
-            height: 110%;
+            height: calc(100% + 1rem);
```

Comment:
Instead of declaring a height, let's remove it, use `bottom` instead and give it the same value as the `top` property.

Reply:
Having `top: -0.1875rem | bottom: -0.1875rem` produced:
![screen shot 2019-02-07 at 1 32 18 pm](https://user-images.githubusercontent.com/8570747/52434420-4416e180-2add-11e9-872c-50c203be0940.png)

Reply:
Make sure that height is auto, not 100%.

Reply:
Thought I had tried that, I don't know why it didn't work yesterday, but it works now. Pushing up now.